packages/aws-cdk/README.md

```diff
@@ -133,6 +133,12 @@ currently deployed stack to the template and tags that are about to be deployed
 will skip deployment if they are identical. Use `--force` to override this behavior
 and always deploy the stack.
 
+In order to pass parameters to your template during deployment, you can use `--parameters
+(STACK:KEY=VALUE)`. This will apply the value `VALUE` to the key `KEY` for stack `STACK`. The stack
+name can either be `*` or omitted in order to apply to all stacks.
+Note: The parameters will not propagate to NestedStacks. These has to be sent with the constructor
```

Comment:
s/These has to/These have to/

or "These must be" would be better

Reply:
agreed, an example with a small snippet of usage would be nice too.